lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -65,6 +83,10 @@ def get_submission_info(self, request, usage_id, submission_uuid):
         """
         data = {'submission_uuid': submission_uuid}
         response = call_xblock_json_handler(request, usage_id, 'get_submission_info', data)
+
+        if response.status_code != 200:
+            raise Exception()
```

Comment:
there are places in these handler where I raised JsonHandlerErrors with, say, 404 error codes. Will those get caught here? Is there some weird handler behavior that hides that? Some of them had error messages that I think would be useful to pass along.

Reply:
This would catch any class of error and suppress error messaging. We _could_ add error information to the error context but I don't think it would actually be used by the frontend since it does its own error display based on BFF error classes. 

Might be helpful to add some sort of logging or analytics so we see what types of errors happen but the value of actually surfacing them to the frontend seems limited.